utils/litemap/src/store/vec_impl.rs

```diff
@@ -98,6 +93,39 @@ impl<K, V> StoreMut<K, V> for Vec<(K, V)> {
     }
 }
 
+impl<K: Ord, V> StoreFromIterable<K, V> for Vec<(K, V)> {
+    fn from_iter_sorted<I: IntoIterator<Item = (K, V)>>(iter: I) -> Self {
```

Comment:
Nit: `lm_from_sorted_iter`

However, down below, you are inserting things into the correct order; you aren't enforcing that the iterator is sorted. I'd therefore name this function `lm_from_iter` or `lm_sort_from_iter`

Reply:
Renamed the method to `lm_sort_from_iter`.